set.mm

```diff
@@ -20160,8 +20160,9 @@ theorem as an axiom of set theory (Axiom 0 of [Kunen] p. 10).  In the
        (Contributed by NM, 14-Sep-2003.)  (Revised by Mario Carneiro,
        6-Oct-2016.)  (Proof shortened by Wolf Lammen, 16-Jun-2019.) $)
     cbvex2 $p |- ( E. x E. y ph <-> E. z E. w ps ) $=
-      ( wn wal wex nfn weq wa notbid cbval2 2exnaln notbii 3bitr4i ) ALZDMCMZLB
-      LZFMEMZLADNCNBFNENUDUFUCUECDEFAEGOAFHOBCIOBDJOCEPDFPQABKRSUAACDTBEFTUB $.
+      ( wex wn wal nfn weq wa notbid cbval2 2nexaln 3bitr4i con4bii ) ADLCLZBFL
+      ELZAMZDNCNBMZFNENUCMUDMUEUFCDEFAEGOAFHOBCIOBDJOCEPDFPQABKRSACDTBEFTUAUB
+      $.
```

Comment:
Ah, yes !  I see how in these four proofs, use of alnex (which is df-ex followed by con2bii) together with con4bi*, instead of df-ex, makes things more streamlined.